src/patch/mod.rs

```diff
@@ -177,6 +181,7 @@ impl Patch for EmbeddedByzantiumPatch {
     fn has_static_call() -> bool { true }
     fn has_revert() -> bool { true }
     fn has_return_data() -> bool { true }
+    fn has_bitwise_shift() -> bool { false }
```

Comment:
This Patch should be `EmbeddedConstantinoplePatch`, or?

Reply:
Oh. Dunno what I did, but my comment just got removed.

Patch name is OK here, but I gotta add an `EmbeddedConstantinoplePatch` indeed

Reply:
Actually decided to remove the `EmbeddedByzantiumPatch`: #353